tasks/zammad_setup.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Download zammad release
+  get_url:
+    url: https://ftp.zammad.com/zammad-latest.tar.gz
+    dest: /tmp/zammad-latest.tar.gz
+    mode: 0440
+
+- name: Create zammad target directory
+  file:
+    path: /opt/zammad
+    state: directory
+    mode: 0775
+
+- name: Unarchive zammad
+  unarchive:
+    src: /tmp/zammad-latest.tar.gz
+    dest: /opt/zammad
+    remote_src: yes
+
+- name: Install base dependencies: bundler, rake and rails
+  gem:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - bundler
+    - rake
+    - rails
+
+- name: Install application dependencies with mysql database support
```

Comment:
Also the name of the task doesn't fit waht it does (mysql vs. postgresql)

Reply:
No. See: "without"-Syntax!
https://docs.zammad.org/en/latest/install-source.html#for-postgresql-note-the-option-says-without-mysql

Reply:
oh I see... :-) but that's weird... :-D

Reply:
I think this is one of the reasons the word "Brainfuck" was invented. I agree, the inverted logic is not good ...